backend/src/main/java/hanglog/trip/domain/DayLog.java

```diff
@@ -47,7 +49,7 @@ public class DayLog extends BaseEntity {
 
     @OneToMany(mappedBy = "dayLog", cascade = REMOVE, orphanRemoval = true)
     @OrderBy(value = "ordinal ASC")
-    private List<Item> items = new ArrayList<>();
+    private Set<Item> items = new HashSet<>();
```

Comment:
```suggestion
    private Set<Item> items = new LinkedHashSet<>();
```

아이템 순서도 보장하게 LinkedHashSet을 사용하는건 어떤가요?

Reply:
`@OrderBy`가 붙여져있으면 내부적으로 LinkedHashSet으로 처리해줘서 그냥 두었는데 명시적으로 LinkedHashSet으로 할당하는 것도 좋겠군여

Reply:
순서는 @OrderBy로 보장하고 있어서 
굳이 HashSet보다 성능이 떨어지는(물론 둘 다 O(1)이긴 하지만) LinkedHashSet로 바꿀 필요까지는 없다는 생각입니다.!

Reply:
내부적으로 LinkedHashSet으로 처리한다면, 어떤 부분에서 성능이 다른가용 ?? 갖고 있을 때 메모리를 더 잡아먹는다 ?? 
저는 명시적으로 + 혹시 모를 버그 예방을 위해 LinkedHashSet에 한 표 하겠습니다 ㅎㅎ

Reply:
`@OrderBy`를 쓰면 HashSet으로 생성을 해도 순서가 보장이 되는지 몰랐네요.

저도 홍고와 라온 말처럼 명시적으로 혹은 혹시 모를 버그를 방지하기 위한 LinkedHashSet으로 할당하는 것에 한표 하겠습니다.

Reply:
성능 차이는 일반 List와 LinkedList의 차이와 같습니다.! 
어떤 자료구조를 사용하냐인데 이건 답글로 일일이 달 필요까지는 없다고 생각합니다.(저희 ArrayList LinkedList 미니 미션도 했으니까요..!)

그리고 명시적으로 라는 부분도 잘 모르곘는게,
Trip의 DayLogs라면 이해가 됩니다. 저희가 추가 삭제 할 때마다 순서에 맞게 기타 DayLog의 실제 위치를 변경했으니까요.!


그런데 DayLog의 Items는 다르다고 생각합니다.
저희가 아이템 위치 바꿀 때 기존에 존재하는 아이템들 사이에 끼워 넣나요?
그냥 order 필드만 바꾸고 있습니다..! 따라서 시간복잡도는 그냥 O(1)이죠
애초에 저희가 순서 변경을 할 때 컬랙션 내에서 item의 위치를 바꾸는게 아닌데
왜 LinkedHashSet으로 바꾸는게 명시적인 효과가 있는건지 잘 모르겠다는 의견입니다..!



그으으으런데 홍고의 말대로 `@OrderBy`가 붙여져있을 때  
HashSet으로 선언해놔도 알아서 LinkedHashSet 형태로 저장되어 있다면 
LinkedHashSet으로 바꿔도 좋을 것 같습니다. (근데 진짜 그렇게 바뀌나요? 그러면 코드 단에서 밑줄로 잡아줘야 하는거 아닌가)